src/game/aiman.h

```diff
@@ -214,7 +214,7 @@ namespace aiman
         else if(ci->aireinit >= 1)
         {
             if(ci->aireinit == 2) loopk(W_MAX) loopj(2) ci->weapshots[k][j].reset();
-            sendf(-1, 1, "ri6si3siv", N_INITAI, ci->clientnum, ci->ownernum, ci->actortype, ci->spawnpoint, ci->skill, ci->name, ci->team, ci->colour, ci->model, ci->vanity, ci->loadweap.length(), ci->loadweap.length(), ci->loadweap.getbuf());
+            sendf(-1, 1, "ri6si3siv", N_INITAI, ci->clientnum, ci->ownernum, ci->actortype, ci->spawnpoint, ci->skill, ci->name, ci->team, ci->colour, ci->model, ci->vanity, ci->loadweap.size(), ci->loadweap.size(), &ci->loadweap);
```

Comment:
On the change from `ci->loadweap.getbuf()` to `&ci->loadweap`, have you tested that this works? When I look in the `sendf` function, it seems to assume that the second argument to the `v` format specifier is an `int` pointer to the internal buffer, not to the `vector` object.

Reply:
Good catch. You have to pass `ci->loadweap.data()`.

Reply:
True that, didn't notice the `int*`